.evergreen.yml

```diff
@@ -230,8 +230,8 @@ github_checks_aliases:
 # Triggered on git tag
 git_tag_aliases:
   - git_tag: "^(\\d+\\.)?(\\d+\\.)?(\\d+)$"
-    variant_tags: [ "release" ]
-    task_tags: [ "image_release", "image_preflight", "openshift_bundles", "code_snippets" ]
+    variant_tags: [ "release", "e2e_smoke_release_test_suite" ]
+    task_tags: [ "image_release", "image_preflight", "openshift_bundles", "code_snippets", "patch-run" ]
```

Comment:
Why does this now include `patch-run` task tags?

Reply:
`patch-run` is required because `e2e_smoke_release_test_suite` variant has tasks with that tag. It will only add tasks matching variant `e2e_smoke_release_test_suite` and not all e2e tests